src/components/Messages/MessageList/livechatCall.js

```diff
@@ -0,0 +1,73 @@
+import { h } from 'preact';
+import { useState } from 'preact/compat';
+
+import { Livechat } from '../../../api';
+import I18n from '../../../i18n';
+import PhoneAccept from '../../../icons/phone.svg';
+import PhoneDecline from '../../../icons/phoneOff.svg';
+import { Avatar } from '../../Avatar';
+import { Button } from '../../Button';
+import { Screen } from '../../Screen';
+import { createClassName, getAvatarUrl } from '../../helpers';
+import { parseDate } from '../MessageTime/index';
+import styles from './styles.scss';
+
+
+const DisplayCallIframe = (rid) => (
+	<Screen.Content nopadding>
+		<div className={createClassName(styles, 'iframe')}>
+			<iframe style='height:100%' src={`${ Livechat.client.host }/meet/${ rid.rid }`} />
+		</div>
+	</Screen.Content>
+);
+
+
+export const ShowCallTime = (props) => (
```

Comment:
I believe this component will no longer be required once [this](https://github.com/RocketChat/Rocket.Chat.Livechat/pull/617/files#r669399143) comment is resolved